sdk/keyvault/keyvault-keys/samples/javascript/cryptography.js

```diff
@@ -20,7 +20,7 @@ async function main() {
   // Connection to Azure Key Vault Cryptography functionality
   let myWorkKey = await client.createKey(keyName, "RSA");
 
-  const cryptoClient = new CryptographyClient(url, myWorkKey.keyMaterial.kid, credential);
+  const cryptoClient = new CryptographyClient(url, myWorkKey.key.kid, credential);
```

Comment:
1. Should `kid` just be `id`?
2. `id` and `name` should be uplifted to `KeyVaultKey`. `version` and `vaultEndpoint` should be left on `properties`.
3. Since those two are to be uplifted, the samples should probably reflect that for ease. I made the same change in the .NET samples so people can see you can get the `id` right from the key.

Reply:
Make changes as necessary throughout. I won't repeat it on other instances below.